themes/ddbasic/sass/components/field.scss

```diff
@@ -6,6 +6,9 @@
 
 .field-type-text-long {
   img {
+    display: inline-block;
+    margin-right: 10px;
+    margin-bottom: 5px;
     &[align="right"] {
       padding: 20px 0 20px 20px;
```

Comment:
This has been changed in new commit.

I noticed that the Media module and File Entity module have been updated since the scss was first written. The scss has therefore been rewritten to match the markup outputted by the file entity module.

Furthermore the filters on the text format have been rearranged, and the title link on the image-file has been removed. This is done to make it possible to add a link-tag around an image in the wysiwyg